canarieapi-cron

```diff
@@ -1,2 +1,2 @@
-* * * * * root python -c 'from canarieapi import logparser; logparser.cron_job()' >> /var/log/logparser_cron.log 2>&1
-* * * * * root python -c 'from canarieapi import monitoring; monitoring.cron_job()' >> /var/log/monitoring_cron.log 2>&1
+* * * * * root [ -n "$CANARIE_PARSE_LOGS" ] && /usr/local/bin/python3 -c 'from canarieapi import logparser; logparser.cron_job()' 2>&1 | /usr/bin/tee -a /var/log/logparser_cron.log >/proc/1/fd/1 2>/proc/1/fd/1
+* * * * * root [ -n "$CANARIE_MONITOR" ] && /usr/local/bin/python3 -c 'from canarieapi import monitoring; monitoring.cron_job()' 2>&1 | /usr/bin/tee -a /var/log/monitoring_cron.log >/proc/1/fd/1 2>/proc/1/fd/1
```

Comment:
Just to make sure I understand what is happening with those redirects. 
Only the logs from `canarie-api-cron` itself will be visible (i.e.: the `logger.info(...)` calls within `parse_log`), and not the logs from `nginx`?

Reply:
yes that's right. This is essentially a way of making cron logs visible on the stdout of the main process (the one running in docker). The problem is that even if you run cron in the foreground with `cron -f` stdout and stderr are still redirected to the cron log files. This is a way of getting around that and sending the logs to actual stdout (as one might expect from a docker container)